batchflow/opensets/imagenette.py

```diff
@@ -32,6 +32,8 @@ class Imagenette(ImagesOpenset):
     """
     SOURCE_URL = 'https://s3.amazonaws.com/fast-ai-imageclas/imagenette.tgz'
     num_classes = 10
+    name_classes = ['tench', 'English springer', 'cassette player', 'chain saw', 'church',
+                    'French horn', 'garbage truck', 'gas pump', 'golf ball', 'parachute']
```

Comment:
Can we add the same attribute for `MNIST`/`CIFAR` and use your `utils.plot_images_predictions` functions in other tutorials as well?